AsyncDisplayKit/ASTableView.h

```diff
@@ -224,100 +211,20 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)waitUntilAllUpdatesAreCommitted ASDISPLAYNODE_DEPRECATED_MSG("Use ASTableNode method instead.");
 
-/**
- * Inserts one or more sections, with an option to animate the insertion.
- *
- * @param sections An index set that specifies the sections to insert.
- *
- * @param animation A constant that indicates how the insertion is to be animated. See UITableViewRowAnimation.
- *
- * @discussion This method must be called from the main thread. The asyncDataSource must be updated to reflect the changes
- * before this method is called.
- */
```

Comment:
I removed the documentation from these methods because they're just overrides of UITableView's methods that don't change behavior.